project/build.scala

```diff
@@ -0,0 +1,77 @@
+import sbt.Keys._
```

Comment:
I will correct the case. Main Cromwell sbt file has compile time dependency specification on logging component therefore moved it in separate Build.scala to support multiple projects with respective sbt file.